packages/components/slideshow/src/js/panel/slideshow-panel.html

```diff
@@ -1,25 +1,25 @@
-<div class="oui-slideshow-panel__picture-container"
-    ng-if="::$ctrl.picture">
-    <div class="oui-slideshow-panel__picture"
+<div class="oui-slideshow-panel__picture-container">
+    <div ng-if="::$ctrl.picture" class="oui-slideshow-panel__picture"
```

Comment:
Not mandatory but we tend to group native attributes and then directives (custom properties).
Could you please move `ngIf` right after the `class` attribute?